File: Python/Mahasiswa.py
# membuat kelas mahasiswa
class Mahasiswa:
    # konstruktor dengan parameter
    def __init__(self, nama, nim, prodi, fakultas):
        self.nama = nama
        self.nim = nim
        self.prodi = prodi
        self.fakultas = fakultas

    # tetapkan nama
    def setNama(self, nama):
        self.nama = nama

    # get nama
    def getNama(self):
        return self.nama
    
    # tetapkan nim
    def setNim(self, nim):
        self.nim = nim
    
    # get nama
    def getNim(self):
        return self.nim
    
    # tetapkan prodi
    def setProdi(self, prodi):
        self.prodi = prodi

    # get nama
    def getProdi(self):
        return self.prodi
    
    # tetapkan fakultas
    def setFakultas(self, fakultas):
        self.fakultas = fakultas
    
    # get nama
    def getFakultas(self):
        return self.fakultas
    
    # tampilkan data
    def tampilkan(self):
        print(self.nama + "\n")
        print(self.nim + "\n")
        print(self.prodi + "\n")
        print(self.fakultas + "\n")

File: PHP/Mahasiswa.php
<?php

// membuat kelas mahasiswa
class Mahasiswa {
    // atribut yang diperlukan
    private $nama = '';
    private $nim = 0;
    private $prodi = '';
    private $fakultas = '';

    // konstruktor dengan parameter
    public function __construct($nama, $nim, $prodi, $fakultas) {
        $this->nama = $nama;
        $this->nim = $nim;
        $this->prodi = $prodi;
        $this->fakultas = $fakultas;
    }

    // tetapkan nama
    public function setNama($nama) {
        $this->nama = $nama;
    }

    // get nama
    public function getNama() {
        return $this->nama;
    }

    // tetapkan nim
    public function setNim($nim) {
        $this->nim = $nim;
    }

    // get nim
    public function getNim() {
        return $this->nim;
    }

    // tetapkan prodi
    public function setProdi($prodi) {
        $this->prodi = $prodi;
    }

    // get prodi
    public function getProdi() {
        return $this->prodi;
    }

    // tetapkan fakultas
    public function setFakultas($fakultas) {
        $this->fakultas = $fakultas;
    }

    // get fakultas
    public function getFakultas() {
        return $this->fakultas;
    }

    // menampilkan data
    public function tampilkan() {
        echo $nama;
        echo $nim;
        echo $prodi;
        echo $fakultas;
    }

    // deconstruct
    function __destruct() {

    }
}
?>
